doc/specs/#1043 - Set the initial position of the Terminal/spec.md

```diff
@@ -0,0 +1,87 @@
+---
+author: Kaiyu Wang KaiyuWang16/kawa@microsoft.com
+created on: 2019-09-03
+last updated: 2019-09-03
+issue id: #1043
+---
+
+# Set the initial position for terminal
+
+## Abstract
+
+This spec is for task #1043 �Be able to set an initial position for the terminal�. It goes over the details of a new feature that allows users to set the initial position and size of the terminal. Expected behavior and design of this feature is included. Besides, future possible follow-up works are also addressed.
+
+## Inspiration
+
+The idea is to allow users to set the initial position of the Terminal when they launch it, prevent the Terminal from appearing on unexpected position (e.g. outside of the screen bounds). We are also going to let users choose to maximize the window when they launch it.
+
+## Solution Design
+
+For now, the Terminal window is put on a default initial position. The program uses CW_USEDEFAULT in the screen coordinates for top-left corner. We have two different types of window � client window and non-client window. However, code path for window creation (WM_CREATE message is shared by the two types of windows) are almost the same for the two types of windows, except that there are some differences in calculation of the width and height of the window.
+
+Four new properties should be added in the json settings file:
+
+**useDefaultInitialPos**: true or false. Determine if we use CW_USEDEFAULT as the initial position of the Terminal window. If true, we ignore initialX and initially. Default is true;

```

Comment:
Do we need this setting? Could the following setting both just be optional settings, and if they exist, assume that for that dimension, we want to use the given position instead of the default?

Reply:
removed. And I also think if the user only provide one axis value, we use system default for the other one ?